scripts/user/orange-csg-to-dot.py

```diff
@@ -133,7 +133,7 @@ def run(infile, outfile, gencls, universe):
         # Load from a .csg.json debug file
         csg_unit = tree[universe]
     else:
-        if isinstance(tree, list):
+        if isinstance(tree, list) and infile is not sys.stdin:
```

Comment:
I think this condition I set was wrong: sorry. Maybe we need to check the contents of the list to see whether it's a ".csg.json" file? Perhaps we should just have the `.csg.json` be a dict that has a format and version? Or for now can we just see if the list's first item is a dict that has "metadata" as a key?

Reply:
yeah, I think for now just checking if the first element in a csg unit dict should be enough